catalyst/dl/callbacks/optimizer.py

```diff
@@ -23,6 +23,7 @@ def __init__(
         accumulation_steps: int = 1,
         optimizer_key: str = None,
         loss_key: str = "loss",
+        decouple_wd = True
```

Comment:
let's use full naming, like `decouple_weight_decay` – it will be much clearer

Reply:
or even `weight_decay_decoupling`?

Reply:
Rename to `decouple_weight_decay`, i like the idea of having verb coming first.